exercises/xorcism/.meta/description.md

```diff
@@ -0,0 +1,81 @@
+Write a streaming adaptor which contains a reference to a key, and bitwise-xors
+it with arbitrary data.
+
+XOR is a fundamental binary operation: for each bit in the inputs, set the
+corresponding bit of the output to `1` if the input bits are different. If both
+inputs are `1` or both are `0`, then the corresponding output bit is `0`.
+
+When XORing a document with a key, the key is repeated as many times as
+necessary, producing an output document equal in length to the input document.
+
+XORing a document with a key has been used for cryptography as recently as the
+early 1900s. While this is thoroughly obsolete as a method for hiding data, it
+can be surprisingly useful for generating noisy random-seeming data without
+needing the complication of true randomness. It is still used occasionally in
+modern cryptography for certain ciphers: the cipher itself is just a mechanism
+for generating a very random, infinitely long key, which is xor'd with the
```

Comment:
sometimes in this problem statement, `xor` is lowercased, but sometimes it is all caps. Is there a rule for when one or the other should be used? If not, might want to settle on one, for internal consistency.

Reply:
I know of no rule, so capitalized all instances in dff3350.